modules/understanding-upgrade-channels.adoc

```diff
@@ -82,6 +83,27 @@ and pre-production and production environments that participate in connected cus
 You can use the `stable-{product-version}` channel to upgrade from a previous minor version of
 {product-title}.
 
+[discrete]
+== eus-{product-version} channel
+
+In addition to the stable channel, certain minor versions of {product-title} offer an
+Extended Update Support[https://access.redhat.com/support/policy/updates/openshift#ocp4_phases] (EUS).
+These EUS versions extend the maintenance phase for customers with Premium Subscriptions
+to 14 months. {product-title} {product-version} is a minor which includes EUS.
+
+While there's no difference between stable-{product-version} and eus-{product-version}
+channels until {product-version} transitions to EUS phase customers may switch
+to the EUS channel as soon as it becomes available. When {product-title} {product-version}
+transitions to the EUS phase of its lifecycle the stable-{product-version} will
+no longer receive updates. Once a cluster upgrades to a version that's exclusive
+to the EUS channel that cluster will no longer be eligible for minor version upgrades
+until upgrades to the next EUS version become available. Currently, upgrades between
+EUS versions will entail a serial set of upgrades, ie: 4.6 to 4.7 to 4.8 to 4.9 to 4.10.
+
+Additionally, you must upgrade to {product-title} {product-version} before it
+transitions to EUS phase, ie: you cannot upgrade from 4.5 to 4.6 after 4.6 transitions
```

Comment:
```suggestion
transitions to EUS phase. For example, you cannot upgrade from 4.5 to 4.6 after 4.6 transitions
```
I'm a little confused about this last paragraph. I feel like there's a bit of context missing, maybe an assumption about the type of channel that you're in before this switchover happens? What happens if you don't upgrade from 4.5 quickly enough? Are you stuck on 4.5 forever? Will the graph let you move from 4.5 to 4.7?

Reply:
The context is that we'll stop testing upgrades from 4.5 to the 4.6 EUS versions. While nothing would prevent them from upgrading from 4.5 to 4.6 on stable channel  then switching to EUS 4.6 channel I don't think we want to encourage that sort of extreme lag. If they wait that long not only has 4.5 gone EOL but 4.6, with the exception of EUS, has too.

This is a bit of an open question that I'm working through with PM but I do assume they're going to be in support of this limitation.